packages/sdk-codegen/package.json

```diff
@@ -33,9 +33,10 @@
     "watch": "yarn lerna exec --scope @looker/sdk-codegen --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments --watch'"
   },
   "dependencies": {
-    "@looker/sdk-codegen-utils": "^21.0.9",
-    "@looker/sdk-node": "^21.0.9",
-    "@looker/sdk-rtl": "^21.0.9",
+    "@looker/sdk": "^21.0.10",
+    "@looker/sdk-codegen-utils": "^21.0.10",
+    "@looker/sdk-node": "^21.0.10",
```

Comment:
we should move this to `devDependencies`

Reply:
Remove `@looker/sdk` and `@looker/sdk-node`. We should only need `@looker/sdk-rtl` hopefully. The `sdk-codegen-utils` package is so incredibly trivial (one small source file) we could just duplicate it for `sdk-codegen` and `sdk-codegen-scripts`.

Reply:
I captured this in #533